### ISSUE TITLE
Data table keyboard navigation

### ISSUE THREAD

**Issue by Einithorynque** [gephi/gephi-lite#229]
Add states for data table's cells. Use for inspiration text button state (same color mixes).

**Comment by sim51** [comment]
See also #211 

> For comfort and accessibility, it should be possible to use keyboard to navigate/edit the data table.
